code/frameworks/nextjs/src/font/webpack/loader/google/get-font-face-declarations.ts

```diff
@@ -1,6 +1,10 @@
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-expect-error
 import loaderUtils from 'next/dist/compiled/loader-utils3';
+import {
+  GoggleFontsDownloadError,
```

Comment:
I think there might be a typo here, Goggle vs Google

Reply:
Well spotted! Thank you <3